app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.kt

```diff
@@ -95,7 +89,13 @@ class TaskDetailFragment : Fragment() {
                 }
             }
         }
-        viewDataBinding.setLifecycleOwner(this.viewLifecycleOwner)
+        viewDataBinding.lifecycleOwner = this.viewLifecycleOwner
+
+        val taskId = arguments?.let {
+            TaskDetailFragmentArgs.fromBundle(it).TASKID
+        }
+        viewDataBinding.viewmodel?.start(taskId)
```

Comment:
just viewModel.start(taskId) (use the instance variable, not the viewmodel from the binding)

Reply:
Done